tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -346,6 +367,22 @@ def sync_volumes(volumes, index, vol_options):
             snapd_status=volumes['volume%s.snapd_svc.online_status' % index],
             snapd_inited=volumes['volume%s.snapd_svc.inited' % index],
         )
+        if not volume.exists():
+            job_id = monitoring_utils.update_dashboard(
+                volume.name,
+                RESOURCE_TYPE_VOLUME,
+                NS.tendrl_context.integration_id,
+                "add"
+            )
+            Event(
```

Comment:
use commons logger utils instead of event

Reply:
i will change this